open-api/rest-catalog-open-api.yaml

```diff
@@ -98,15 +98,13 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/CatalogConfiguration'
+                $ref: '#/components/schemas/CatalogConfig'
               example: {
-                "data": {
```

Comment:
This top-level `data` object field is no longer used in the responses, but wasn't updated for this one example.

Reply:
Screenshot of the new example in the swagger UI.

![image](https://user-images.githubusercontent.com/9833362/163243752-6e831bbc-4898-4ee6-8419-3d81d6f33557.png)